chain/chain/src/store.rs

```diff
@@ -229,6 +224,11 @@ pub trait ChainStoreAccess {
         &mut self,
         hash: &CryptoHash,
     ) -> Result<&ExecutionOutcomeWithIdAndProof, Error>;
+    /// Returns a HashSet of Outcome ids for current Block Hash
+    fn get_all_block_hash_to_outcomes(
```

Comment:
This name is confusing. What about `get_outcomes_by_block_hash`?

Reply:
++